interactions/models/discord/embed.py

```diff
@@ -207,10 +207,10 @@ class Embed(DictSerializationMixin):
         repr=False, default=None, converter=c_optional(EmbedAttachment.from_dict)
     )
     """The thumbnail of the embed"""
-    image: Optional[EmbedAttachment] = attrs.field(
```

Comment:
We should be keeping this for schema consistency

Reply:
Why? It has no affect on the final output, and opens up this feature? Not to mention this entire version is breaking